src/main/java/org/kuali/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -1468,6 +1474,13 @@ protected boolean isTaxReviewRequired() {
      * @return true if the payee is a vendor and has withholding dates therefore should receive tax review, false otherwise
      */
     protected boolean isPayeePurchaseOrderVendorHasWithholding() {
+        /* CU Customization: Null checks added to prevent NPE when payee is not selected on saved DV causing buttons to not present for a super user.
+         */
+        if (ObjectUtils.isNull(this.getDvPayeeDetail()) || 
+                ObjectUtils.isNull(this.getDvPayeeDetail().getDisbursementVoucherPayeeTypeCode())) {
```

Comment:
There's some code in the isTaxReviewRequired() method (both in the overlay and in the CU subclass) that expects getDvPayeeDetail() and getDisbursementVoucherPayeeTypeCode() to return non-null values.

I also just noticed something. It appears that DisbursementVoucherDocument and CuDisbursementVoucherDocument have separate "dvPayeeDetail" instance variables, so perhaps only one of them is being populated properly due to our getter/setter overrides. Could CuDisbursementVoucherDocument be updated to use the same "dvPayeeDetail" variable as the parent class (or, alternatively, to change the setter method setup to update both variables)?